src/data/SeriesUtils.tsx

```diff
@@ -284,18 +287,35 @@ function buildSeries(
   let sum = 0;
   let remainder = 0;
   let binIndex = 0;
+  let nullCount = 0;
+  let nullRemainder = 0;
+  let isNull = false;
 
   for (let index = 0; index < values.length; index++) {
     sum += remainder;
+    nullCount += nullRemainder;
+
+    isNull = values[index] == null;
+
     if (index >= binIndex * binSize && index + 1 <= (binIndex + 1) * binSize) {
       // The value is completely in the bin
       sum += values[index];
       remainder = 0;
+      nullRemainder = 0;
+
+      if(isNull){nullCount += 1;}
```

Comment:
```suggestion
      if (isNull) {
        nullCount += 1;
      }
```

Reply:
@boazreicher I'm curious how you generated this suggestion.  Did you run a linter in GitHub somehow?  What linter do you prefer?

Reply:
I just added it manually in the Github UI.  Locally I usually use [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier)